src/main/java/wooteco/subway/line/controller/LineController.java

```diff
@@ -0,0 +1,59 @@
+package wooteco.subway.line.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import wooteco.subway.line.dto.LineRequest;
+import wooteco.subway.line.dto.LineResponse;
+import wooteco.subway.line.service.LineService;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+public class LineController {
+    private static final Logger log = LoggerFactory.getLogger("console");
+
+    private final LineService lineService;
+
+    public LineController(LineService lineService) {
+        this.lineService = lineService;
+    }
+
+    @PostMapping("/lines")
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        LineResponse newLine = lineService.save(lineRequest);
+        log.info("An INFO Message : {}", newLine.getName() + " 노선 생성 성공");
+        return ResponseEntity.created(URI.create("/lines/" + newLine.getId())).body(newLine);
+    }
+
+    @GetMapping(value = "/lines", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<LineResponse>> showLines() {
+        List<LineResponse> allLines = lineService.findAllLines();
+        log.info("An INFO Message : {}", "지하철 모든 노선 조회 성공");
+        return ResponseEntity.ok().body(allLines);
+    }
+
+    @GetMapping(value = "/lines/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<LineResponse> showLine(@PathVariable Long id) {
+        LineResponse line = lineService.findLine(id);
```

Comment:
```suggestion
        LineResponse line = lineService.findBy(id);
```
어떤 조건으로 검색했는지 메서드명에 By를 붙여서 알려주세요
그리고 LineService니까 당연히 Line 객체에 대한 처리를 하지 않을까요? 메서드에 중복으로 어떤 객체를 반환하는지 표시할 필요가 있을까요?